frontend/src/domain/Account/Orders/Order/Order.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import Collapsible from 'react-collapsible';
+import * as P from './parts';
+import Arrow from '../../../../assets/arrow.svg';
+import User from '../../../../assets/user.svg';
+
+interface TempProduct {
+   id: string;
+   name: string;
+   price: string;
+   quantity: number;
+}
+
+interface OrderProps {
+   order: {
+      id: string;
+      date: string;
+      products: TempProduct[],
+      delivery: string;
+      deliveryCost: string;
+      payment: string;
+      paymentCost: string;
+      status: string;
+      total: string;
+      basketPrice: string;
+      deliveryData: {
+         recipient: string;
+         email: string;
+         phone: number;
+         street: string;
+         city: string;
+         zipCode: string;
+      }
+   }
+};
+
+export const Orders: React.FC<OrderProps> = ({ order }) => {
+   const [isOpen, setIsOpen] = useState(false);
+
+   const Trigger = (
+      <P.TriggerWrapper>
+         <P.Text>{order.id}</P.Text>
+         <P.Text>{order.date}</P.Text>
+         <P.Text>{order.delivery}</P.Text>
+         <P.Text>{order.payment}</P.Text>
+         <P.Text>$ {order.total}</P.Text>
+         <P.Text>{order.status}</P.Text>
```

Comment:
Can we extract Trigger from Orders comp, receive order as prop and iterate through object properties? e.g.
```
cont triggerProps = ['id', 'date', 'delivery', 'payment', 'total', 'status'];
...
Object.keys(order).filter(triggerProps.contains).map(key => <P.Text key={key}>{order[key]}</P.Text >
```

Reply:
Moved as as separate component, but this component is so simple that doing object keys, filtering and mapping make it a bit unreadable